go.mod

```diff
@@ -35,7 +35,7 @@ require (
 	github.com/protocolbuffers/txtpbfmt v0.0.0-20220608084003-fc78c767cd6a
 	github.com/yoheimuta/go-protoparser/v4 v4.9.0
 	golang.org/x/crypto v0.18.0
-	golang.org/x/exp v0.0.0-20240119083558-1b970713d09a
+	golang.org/x/exp v0.0.0-20230905200255-921286631fa9
```

Comment:
I'm not clear why this version is going backwards. Probably doesn't hurt anything, but it's curious and I wanted to flag it.

Reply:
I am a bit confused how it got updated to begin with - the dependency graph shows the older version being the newest requirement when running `go mod tidy`.  I reverted these lines.